tests/nvme_copy_test.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
+#
```

Comment:
I think we should start using SPDX licensee identifier for all files and remove the license text. Would you mind submitting these new files with a SPDX tag?

The main motivation is to make any tooling around tool chain verification simpler.

Reply:
@igaw you mean to say by adding "SPDX-License-Identifier: LGPL-2.1-or-later" this text and remove "Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved." this one right ?